core/server/api/shared/frame.js

```diff
@@ -37,6 +37,10 @@ class Frame {
             }
         }
 
+        if (apiConfig.rawInput) {
+            this.options.rawInput = apiConfig.rawInput;
```

Comment:
Even if the shared serializer would transform `include` to `withRelated`, you always have access to `frame.original.include` or? 🤔